Gopkg.toml

```diff
@@ -40,7 +40,7 @@
 
 [[constraint]]
   name = "github.com/giantswarm/micrologger"
-  branch = "master"
+  version = "v0.1.0"
```

Comment:
Do we need to pin `microerror` as well? And from what I understand, `v0.1.0` is equivalent to `^v0.1.0` which is equivalent to `>= v0.1.0, < 1.0.0`. I guess I'm not sure if it should be an exact version (`=`), minor range (`~`), or major range (`^`).

Reply:
Oh sorry! You just commented when I was merging

Reply:
[This looks like `v0.1.0`](https://github.com/giantswarm/k8sclient/tree/master/vendor/github.com/giantswarm/micrologger) , even though there is v0.2.0 already released, so I believe we are fine?

Reply:
Yes, I guess that's alright. What about `microerror` here?